contents.json

```diff
@@ -18337,6 +18337,24 @@
       "date_added": "Oct 26 2019",
       "suggested_by": "@lucanaef",
       "stars": 0
+    },
+    {
+      "title": "Exa Wallet - Monero",
+      "category-ids": ["cryptocurrency"],
+      "description": "Receive, spend and store your Monero with an open-source multisignature wallet.",
```

Comment:
Could you remove the period after wallet? We do not generally need it on this page. Thx